rpc/js/src/peer.ts

```diff
@@ -3,9 +3,29 @@ interface ReadyPeer {
   dc: RTCDataChannel;
 }
 
+export function addCustomSdpFields(
+  sdpFields?: object,
+  localDescription?: RTCSessionDescription | null
+) {
+  let description = {
+    sdp: localDescription?.sdp,
```

Comment:
if localDesc is null, does this work?, what does `localDescription?.sdp` evaluate to?

Reply:
`localDescription?.sdp` would return undefined, which would then be `description.sdp = undefined`. The concatenation would still work as intended.